lib/ipaddr.h

```diff
@@ -165,8 +166,14 @@ static inline bool ipaddr_is_zero(const struct ipaddr *ip)
 	return true;
 }
 
+static inline bool ipaddr_is_same(const struct ipaddr *ip1,
+				  const struct ipaddr *ip2)
+{
+	return (ipaddr_cmp(ip1, ip2) == 0);
+}
+
 #ifdef _FRR_ATTRIBUTE_PRINTFRR
-#pragma FRR printfrr_ext "%pIA"  (struct ipaddr *)
+#pragma FRR printfrr_ext "%pIA"(struct ipaddr *)
```

Comment:
AFAIR, this should be as before (whitespace is required before `(struct ...)`) / (ignore styling here).

Or better wrap this at all into:

```
/* clang-format off/on */
```

Reply:
I've wrapped the block in the `/* clang-format off/on */` block and reverted the change to the line.